lib/tasks/projects.rake

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+namespace :projects do
+  desc "Times the creation of projects and querying by TigerData metadata fields"
+  task :create_many, [:count, :prefix] => [:environment] do |_, args|
+    count = (args[:count] || "").to_i
```

Comment:
I'm sure the code you have works, but it would read better to put `raise "Count must be specified" if count.blank?` and then you would not have to deal with the empty string parsing
```suggestion
    raise "Count must be specified" if count.blank?
    count = args[:count].to_i
```